llvm/lib/Transforms/IPO/SampleProfile.cpp

```diff
@@ -444,15 +444,42 @@ class SampleProfileMatcher {
   // the profile.
   StringMap<LocToLocMap> FuncMappings;
 
-  // Profile mismatching statstics.
+  // Match state for an anchor/callsite.
+  enum class MatchState {
+    Unknown = 0,
+    // Initial match in pre-match.
```

Comment:
For all instances: 

"in pre-match" -> "between input profile and current IR"

"in post-match" -> "after fuzzy profile matching"